### ISSUE TITLE
Update membership list to accurately reflect members

### ISSUE THREAD

**Issue by joseph-mueller** [IviFoundation/ivifoundation.github.io#36]
Per title, I think the current info may be stale, especially general members
http://localhost:4000/membership/current_members.html

**Comment by alejandro5042** [comment]
Makes sense! Let's also assign issues to an appropriate milestone, unless it's tech debt that doesn't fit in one.

![image](https://user-images.githubusercontent.com/3143819/221375719-b0d53758-d155-4587-8fcb-dca9f562d50e.png)

Note: We can always prioritize later if we end up with too many.

**Comment by GabeNI** [comment]
Updated some info for NI & Keysight in #113
